framework/include/parser/Parser.h

```diff
@@ -154,6 +176,10 @@ class Parser
 
   /// The input file names
   const std::vector<std::string> _input_filenames;
+
   /// The optional input text (to augment reading a single input with the MooseServer)
   const std::optional<std::string> _input_text;
+
+  /// The app types extracted from [Application] block
```

Comment:
```suggestion
  /// The application type extracted from [Application] block
```
there is only one right